src/frameworks/eleventy.json

```diff
@@ -8,8 +8,11 @@
   },
   "watch": {
     "command": "npx @11ty/eleventy --watch",
-    "directory": "_site",
     "port": 8080
   },
+  "build": {
+    "command": "npx @11ty/eleventy",
```

Comment:
Is `npx` necessary?
People seem to document `npx` mostly to be able to run locally installed binaries. However, libraries like [`npm-run-path`](https://github.com/sindresorhus/npm-run-path) (which is used by `execa` when using the `preferLocal: true` option) fix that problem. For example, other frameworks should have the same problem.

Reply:
You're right. This is just me trying to be consistent with https://www.11ty.dev/docs/usage/ but since the detector checks for the local dependency and `preferLocal: true` we don't need it.
Done in 76adf9b (also fixes another issue introduced by https://github.com/netlify/framework-info/pull/74).
We should only use `--serve` (`--watch` watches file but don't start a dev server).